lib/glific/messages.ex

```diff
@@ -759,4 +759,33 @@ defmodule Glific.Messages do
     Message
     |> struct(opts)
   end
+
+  @doc """
+  Get session uuid of contact's last inbound message
+  """
+  @spec get_session_uuid(map()) :: Ecto.UUID.t()
+  def get_session_uuid(message_params) do
+    organization = Partners.organization(message_params.organization_id)
+
+    with {:ok, contact} <-
```

Comment:
I think we already have a function called get organization contact which avoided the DB call?

Reply:
We might remove this code and avoid DB call when switching to trigger.